workloads/hyper-scale/env.sh

```diff
@@ -13,8 +13,20 @@ export NUMBER_OF_HOSTED_CLUSTER=${NUMBER_OF_HOSTED_CLUSTER:-2}
 export COMPUTE_WORKERS_NUMBER=${COMPUTE_WORKERS_NUMBER:-24}
 
 # Hosted cluster spec
+export HYPERSHIFT_OPERATOR_IMAGE="quay.io/hypershift/hypershift-operator:latest"
 export RELEASE_IMAGE=""
```

Comment:
User should be able the release image and others using environment variables like @mukrishn and I discussed.